specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -186,6 +186,137 @@
           }
         }
       }
+    },
+    "/analyze-conversation/jobs": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "description": "Submit a long conversation for analysis. Specify one or more unique tasks to be executed as a long-running operation.",
```

Comment:
nit: remove `long`

Reply:
Use the same description we have in `analyze-text`, just replace `text` with `conversation`.
`"description": "Submit a collection of text documents for analysis. Specify one or more unique tasks to be executed."`

Reply:
will update,

Reply:
Merged into conversationAnalysis.json